libs/wingsdk/src/target-tf-gcp/bucket.inflight.ts

```diff
@@ -0,0 +1,224 @@
+import { Storage, Bucket } from "@google-cloud/storage";
+import { BucketDeleteOptions, IBucketClient, SignedUrlOptions } from "../cloud";
+import { Json } from "../std";
+
+export class BucketClient implements IBucketClient {
+  private _public: boolean;
+  private bucketName: string;
+  private storage: Storage;
+  private bucket: Bucket;
+
+  constructor(
+    bucketName: string,
+    isPublic: boolean = false,
+    storage: Storage,
+    projectId: string = process.env.GCP_PROJECT_ID || "wingsdk-test"
```

Comment:
I wonder if it can lead to conflicts, since we can also specify the projectId in [AppProps](https://github.com/winglang/wing/blob/main/libs/wingsdk/src/target-tf-gcp/app.ts#L17)

Reply:
In this scenario, it will accept the props first if it is passed. if not, then it will check the env for `GCP_PROJECT_ID` and if still it didn't find anything it will gonna use `wingsdk-test` as default.

I'm doing it this way because during the test at test/shared-gcp/bucket-inflight.test.ts we're defining a mockeStorage bucket instance without projectId

Reply:
Sounds cool, thank you for the clarification :)